app/javascript/components/ParticipantShowPage/index.js

```diff
@@ -43,16 +51,77 @@ class ParticipantShowPage extends React.Component {
       participantId,
       personalQuestionnaire,
       professionalQuestionnaire,
+      userType,
     } = this.props;
 
     return (
       <ThemeProvider theme={theme}>
-        <Grid container direction="row">
-          <Grid item xs={6} className={classes.leftHalf}>
+        <Grid
+          container
+          direction="row"
+          style={{ height: '100vh', width: '100vw' }}
+        >
+          <Grid item xs={1} className={classes.navBar}>
+            <Grid container direction="column" alignItems="center" spacing={5}>
+              <Grid item>
+                <Button
+                  component="a"
+                  disableFocusRipple
+                  disableTouchRipple
+                  className={classes.navBarSignOut}
+                >
+                  Sign Out
+                </Button>
+              </Grid>
+              <Grid item>
+                <IconButton
+                  disableFocusRipple
+                  disableTouchRipple
+                  className={classes.navBarItem}
+                >
+                  <HomeIcon fontSize="large" />
+                </IconButton>
+              </Grid>
+              <Grid item>
+                <Grid
+                  item
+                  xs
+                  container
+                  justify="center"
+                  direction="column"
+                  spacing={10}
+                >
+                  {/* Grid items used to pad Unloop logo to bottom of screen */}
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
+                  <Grid item>{/* Intentionally left blank */}</Grid>
```

Comment:
I would instead set the Grid container that surrounds the entire navbar to be `justify="space-between"` and surround a Grid item around the actions and another around the logo